### DISCUSSION TITLE
The latest Pro version

### DISCUSSION THREAD

**Discussion by Osushi** [omkarcloud/google-maps-scraper#216] [Q&A]
I am downloading from the purchase Github link to install the latest pro version.

![スクリーンショット 2024-10-15 16 15 58](https://github.com/user-attachments/assets/9e9f79b7-84e3-4d49-ac32-bf8f2c3696ce)

At some point, this link stopped working.
How can I download the latest version?

**Comment by Chetan11-dev** [comment]
Could you mail me, so I can give you the last python node version?
